paasta_tools/cli/cmds/mark_for_deployment.py

```diff
@@ -979,6 +985,23 @@ def after_state_change(self):
         self.update_slack()
         super().after_state_change()
 
+    def individual_slo_callback(self, label, bad):
+        if bad:
+            message = f"SLO started failing: {label}"
+        else:
+            message = f"SLO stopped failing: {label}"
```

Comment:
Can you rephrase this so it isn't a double-negative?
How about

    The {label} SLO is now OK.

Reply:
> This looks great as a first pass.
> I'm a tad worried about failures with sfx, exceptions, etc.
> How do you feel about making those threads "bulletproof" so that some sort of exception in them cannot possibly make mfd crash?

Exceptions in threads don't bubble up to the main thread, so we should be okay there.